webapp/views.py

```diff
@@ -149,15 +149,16 @@ def create_asset():
     friendly_name = request.values.get("friendly-name")
     optimize = request.values.get("optimize", False)
     tags = request.values.get("tags", "")
+    tags = re.split(",|\\s", tags)
```

Comment:
why do we use `re.split` here and not in the `update_asset` view?

Reply:
I wasn't sure about this, I removed the split by space as there are some tags that are multiple words separated by space...